src/hearings/containers/hearing-actuals/hearing-stage-result/hearing-stage-result.component.ts

```diff
@@ -45,13 +46,15 @@ export class HearingStageResultComponent implements OnInit, OnDestroy {
       adjournedReason: [''],
       cancelledReason: ['']
     });
-    this.sub = this.hearingStore.select(fromHearingStore.getHearingActuals)
+    this.sub = combineLatest([this.hearingStore.select(fromHearingStore.getHearingActuals), this.route.paramMap])
       .pipe(
-        filter((state: HearingActualsStateData) => !!state.hearingActualsMainModel)
+        filter(([state]: [HearingActualsStateData, ParamMap]) => !!state.hearingActualsMainModel),
+        first()
       )
-      .subscribe((state: HearingActualsStateData) => {
-        this.hearingActuals = state.hearingActualsMainModel;
-        this.hearingStageResultForm.get('hearingStage').setValue(this.hearingActuals.hearingPlanned.plannedHearingType);
+      .subscribe(([state, params]: [HearingActualsStateData, ParamMap]) => {
+        this.id = params.get('id');
```

Comment:
this this.id = params.get('id'); could be put into the constructor to load the id, no need to use combineLatest.

Reply:
The loading of id is now in the constructor and removed the use of combineLatest.